packages/evo/src/agent-functions/executeScript.ts

```diff
@@ -114,6 +114,9 @@ export const executeScript: AgentFunction<AgentContext> = {
           return ResultOk(EXECUTE_SCRIPT_ERROR_RESULT(params.namespace, SCRIPT_NOT_FOUND(params), params));
         }
 
+        // if we are executing a script, it means a script was created or found
+        context.chat.condenseFindScriptMessages(params.namespace);
```

Comment:
wouldn't it be possible to call this in this line https://github.com/polywrap/evo.ninja/blob/15da04ccc362006fec7db52f5f1a2b5d142ccf08/packages/agent-utils/src/agent/basicFunctionCallLoop.ts#L27 cc @nerfZael

Reply:
Yes, with some modifications to obtain the executed function info, that is possible!

Reply:
I made the adjustment. It is still coupled to the scripts, in the sense that a change to the scripts' message contents will require us to change the `condensFindScriptMessages` method. But this is definitely an improvement.